README.md

```diff
@@ -1 +1,24 @@
-# divinecharlotte-hello-rails-react
\ No newline at end of file
+# README
+
+This README would normally document whatever steps are necessary to get the
+application up and running.
+
+Things you may want to cover:
+
+* Ruby version
+
+* System dependencies
+
+* Configuration
+
+* Database creation
+
+* Database initialization
+
+* How to run the test suite
+
+* Services (job queues, cache servers, search engines, etc.)
+
+* Deployment instructions
+
+* ...
```

Comment:
- [ ] Kindly update your readme file to a more professional readme instead of using the RoR default template. you can use [this readme template](https://github.com/microverseinc/readme-template) provided by Microverse and please remember to edit each section to better suit the project.